### PR TITLE
Proper display / filtering of ROOT log messages from stderr in terminal output and InfoLogger

### PR DESCRIPTION
**Pull request by ehellbar**
Applying the same logic for the terminal/GUI output and the InfoLogger.

For the terminal output, there is a new `logLevel` variable in the `DeviceInfo` which is initialized to the severity level of the device from the command line or to `Info` by default. This variable is independent of the `DeviceControl::logLevel` and it replaces the `DeviceControl::logLevel` to determine which logs are printed in the terminal. Like this, the `DeviceControl::logLevel` only sets the severity of the logs displayed in the GUI itself.

### CODE REVIEW

File: Framework/Core/include/Framework/LogParsingHelpers.h [hunk 1]
```diff
@@ -34,6 +35,9 @@ struct LogParsingHelpers {
   /// Available log levels
   static char const* const LOG_LEVELS[(int)LogLevel::Size];
 
+  /// Map from ROOT log string to log level
+  static const std::map<std::string, LogLevel> MAP_ROOT_LOG_LEVELS;
```
Comment (ktf) [inline Framework/Core/include/Framework/LogParsingHelpers.h]: std::map is really for sorted lookups where you keep adding and removing from the map. Maybe you should use a std::unordered_map. In any case, please move it out of the header.

Comment (ehellbar) [reply Framework/Core/include/Framework/LogParsingHelpers.h]: removed this map, used an if...else block instead like for the Fair logs.

**Review by davidrohr — APPROVED** [review]
Looks good. Just one question: What will happen with severity unknown? Will it be filtered, e.g. if we say --severity error, or will it be printed?

**Comment by ehellbar** [comment]
severity unknown is numerically bigger than fatal, so it will always be printed. And in the InfoLogger, it will be set to error, which is (and was) the default severity for stderr messages.

**Pull request merged**